sdk/identity/identity/README.md

```diff
@@ -233,6 +240,10 @@ setLogLevel("info");
 
 ### Read the documentation
 
+The client libraries listed [here][releases] support authenticating with `TokenCredential` and the Azure Identity library. You can learn more about their usage and find additional documentation along with samples in the links mentioned [here][releases]
+
+The [microsoft-graph-sdk][graphsdk] also supports authenticating with `TokenCredential` and the Azure Identity library.
```

Comment:
Wowza! Didnt realize this had happened.

Regardless, I am worried about listing packages that support TokenCredential as this list will always be increasing. Any reason to particularly call the packages out rather than rely on the readme of those packages to say whether they support credentials from our package or not?